src/props/onApprove.js

```diff
@@ -148,6 +147,13 @@ type OrderActionOptions = {|
 
 function buildOrderActions({ intent, orderID, restart, facilitatorAccessToken, buyerAccessToken, partnerAttributionID, forceRestAPI, onError, experiments } : OrderActionOptions) : OrderActions {
     const get = memoize(() => {
+        if (experiments?.upgradeLSATWithIgnoreCache) {
+            return upgradeFacilitatorAccessTokenWithIgnoreCache(facilitatorAccessToken, buyerAccessToken, orderID)
+                .then((res) => {
+                    facilitatorAccessToken = res;
+                    return getOrder(orderID, { facilitatorAccessToken, buyerAccessToken, partnerAttributionID, forceRestAPI, experiments });
```

Comment:
I'd suggest the following to make it a little clearer what is happening:
```suggestion
                .then((upgradedFacilitatorAccessToken) => {
                    return getOrder(orderID, { facilitatorAccessToken: upgradedFacilitatorAccessToken, buyerAccessToken, partnerAttributionID, forceRestAPI, experiments });
```